package.json

```diff
@@ -22,11 +22,11 @@
     "lint": "eslint --quiet src test",
     "lint:fix": "eslint --quiet --fix src test",
     "prepare": "husky install",
-    "test": "npm run test:unit && npm run test:integ",
+    "test": "npm run test:unit",
     "test:unit": "mocha test/unit/**/*.spec.js --timeout=3000",
     "test:integ": "mocha test/integration/**/*.spec.js --timeout=3000",
     "printReportsLink": "echo Detailed unit test coverage report: file:///$(pwd)/coverage-unit/index.html && echo Detailed integration test coverage report: file:///$(pwd)/coverage-integration/index.html",
-    "cover": "npm run cover:unit && npm run cover:integ",
```

Comment:
because for now there is no index file (which was returning the 'hello world yo' as a string ) and we were checking it .
In incoming pull requests index.js file will be introduced and with that these "`cover`" command will again be restored back to both unit and integ.